py4web/utils/auth.py

```diff
@@ -1664,7 +1664,7 @@ def profile(self, model=False):
                         deletable=deletable)
 
         form = Form(
-            fields,
+            self.auth.db.auth_user,
```

Comment:
why this change?

Reply:
Hi Massimo, this was something I broke with my previous Auth API updates.

The commit fixes issue #697 where the profile details cannot be updated currently due to behaviour in the Form class setting the dbio flag to false if the value passed to the class is a list object rather than the actual table object.